bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WH_WToLNu_1j_SMEFTsim_topU3l/WH_WToLNu_1j_SMEFTsim_topU3l_run_card.dat

```diff
@@ -0,0 +1,271 @@
+#*********************************************************************
+#                       MadGraph5_aMC@NLO                            *
+#                                                                    *
+#                     run_card.dat MadEvent                          *
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#*********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  WH_WToLNu_LO_GP_gen     = run_tag ! name of the run 
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  .true.     = gridpack  !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  5000000 = nevents ! Number of unweighted events requested 
```

Comment:
What is the reason for a large pilot run?

Reply:
Changed to 1.e+5. 

IIRC, we need to produce 1% of total events in the pilot run, so this is the choice.

Reply:
1% of the total number of events you are going to produce? why? 
the pilot run will not have any affect on the pre-sampling or precision.

Reply:
Okay, then it should not matter. 

I thought this is the number of events used for phase-space sampling in gridpack. But that's not true, I'm fine to reduce it even further if you wish.